changelog_unreleased/cli/12800.md

```diff
@@ -0,0 +1,19 @@
+#### Add `--cache` CLI option (#12800 by @sosukesuzuki)
+
+Two new CLI options have been added for a caching system similar to [ESLint's one](https://eslint.org/docs/user-guide/command-line-interface#caching). Please see the [documentation](https://prettier.io/docs/en/cli.html#--cache) for details.
+
+##### `--cache`
+
+When you want to format only changed files, you can run Prettier with `--cache` flag. Enabling this flag can dramatically improve running time.
```

Comment:
This sounds like CLI will skip unchanged files, but it's not.

Reply:
What is wrong?

Reply:
Even if file is unchanged, but last run was not using `--cache` we'll format it.

Reply:
Or only config changed, but file didn't change.

Reply:
I got it. However, I have no idea to better sentence now... I'll figure it out later.